app/components/treatment/voting/voter-row.hbs

```diff
@@ -0,0 +1,30 @@
+<tr>
+  {{#if this.fetchPersoon.lastSuccessful}}
+  <td>
+    {{this.persoon.gebruikteVoornaam}} {{this.persoon.achternaam}}
+  </td>
+  <td>
+    {{@row.[0].bekleedt.bestuursfunctie.label}}
+  </td>
+  <td>
+    {{#if @manageVotes}}
+    <PowerSelect @searchEnabled={{false}} @options={{this.voteOptions}} @selected={{this.selectedVote}}
+      @onchange={{this.onSelectVote}} as |singleselect|>
```

Comment:
You can fix the PowerSelect bug by using "onChange" instead of "onchange" I believe.
```
<PowerSelect @searchEnabled={{false}} @options={{this.voteOptions}} @selected={{this.selectedVote}}
      @onChange={{this.onSelectVote}} as |singleselect|>
```

Reply:
I tried onChange first but it didnt work
Ill try to see if the WU version works better